### PR TITLE
[WIP] Copy address to clipboard when using next locally

### PR DESCRIPTION
**Pull request by darahak**
I liked this little feature when I used [serve](https://github.com/zeit/serve), so I thought it would be nice to have it on Next too.

### CODE REVIEW

File: bin/next-dev [hunk 1]
```diff
@@ -60,7 +61,9 @@ const srv = new Server({ dir, dev: true })
 srv.start(argv.port, argv.hostname)
 .then(async () => {
   if (!process.env.NOW) {
-    console.log(`> Ready on http://${argv.hostname ? argv.hostname : 'localhost'}:${argv.port}`)
+    const url = `http://${argv.hostname ? argv.hostname : 'localhost'}:${argv.port}`
+    copySync(url)
```
Comment (timneutkens) [inline bin/next-dev]: Lets do it the same way as micro-dev https://github.com/zeit/micro-dev/blob/3d264d50dc0d30ea88a9563287e020a112e67365/lib/listening.js#L13-L20

Comment (darahak) [reply bin/next-dev]: @timneutkens Is it okay if I put this `copyToClipboard` in a new `bin/utils.js`?
If I export it from `lib/utils.js`, I get this error when running `next`:

```
 ERROR  Failed to compile with 6 errors                                                                                                                                               17:38:21

These dependencies were not found:

* child_process in D:/Projects/next.js/node_modules/cross-spawn/index.js, D:/Projects/next.js/node_modules/execa/index.js
* fs in D:/Projects/next.js/node_modules/cross-spawn/lib/util/readShebang.js, D:/Projects/next.js/node_modules/isexe/index.js and 2 others

To install them, you can run: npm install --save child_process fs
```

File: bin/next-start [hunk 2]
```diff
@@ -50,7 +51,9 @@ const srv = new Server({ dir })
 srv.start(argv.port, argv.hostname)
 .then(() => {
   if (!process.env.NOW) {
-    console.log(`> Ready on http://${argv.hostname ? argv.hostname : 'localhost'}:${argv.port}`)
+    const url = `http://${argv.hostname ? argv.hostname : 'localhost'}:${argv.port}`
+    copySync(url)
+    console.log(`> Ready on ${url} (copied on clipboard)`)
```
Comment (tusbar) [inline bin/next-start]: "copied **to** clipboard" maybe?

Comment (darahak) [reply bin/next-start]: Yep. I already changed it.

**Comment by zenflow** [comment]
Nooo.. sometimes I have things saved to my clipboard I don't want lost

**Review by timneutkens — CHANGES_REQUESTED** [review]
Serve has an option to disable this behaviour https://github.com/zeit/serve/blob/3af40d3315378588c3ae22d3b821209fa480a17d/lib/options.js#L39-L43 please add it 👍

**Comment by timneutkens** [comment]
Going to close this for now, feel free to open a new pull request with the opt-out ❤️

**Pull request closed without merging**